### PR TITLE
Loot allocation changes

### PR DESCRIPTION
**Pull request by antonypap**
TODO:

1. sort agents into rep order and access that through main for all funcs in loot stage
2. for each agent ask what item/s they want the most
3. give them these item(s) (multiple items?)
4. repeat this until all loot is gone, then stop

### CODE REVIEW

File: pkg/infra/main.go [hunk 1]
```diff
@@ -223,6 +223,7 @@ func startGameLoop() {
 				logging.OutputLog(logging.Loss)
 
 				csvFile.Close()
+				fmt.Println(csvFile.Name())
```
Comment (MattSScott) [inline pkg/infra/main.go:226]: deliberately removed this since all have same name

Comment (antonypap) [reply pkg/infra/main.go:226]: Hm not really (although that might seem a bit ambiguous https://pkg.go.dev/os#Create), i just added that for the sake of completeness in order to use the `runtest.sh`

File: pkg/infra/game/stage/loot/loot.go [hunk 2]
```diff
@@ -89,21 +90,17 @@ func AgentLootDecisions(
 		start <- startLootMessage
 	}
 
+	fmt.Println("Starts all sent")
+
```
Comment (antonypap) [inline pkg/infra/game/stage/loot/loot.go:102]: Nice change, make sure to rm the print afterwards before merging!

File: pkg/infra/main.go (hunk 1 above)
Comment (MattSScott) [reply pkg/infra/main.go:226]: I named them all output.csv for post-processing

File: pkg/infra/game/stage/loot/loot.go [hunk 3]
```diff
@@ -200,7 +204,7 @@ func HandleLootAllocationExhaustive(globalState state.State, pool *state.LootPoo
 	return &globalState
 }
 
-func removeItemFromList(item state.Item, itemList []state.Item) (error, []state.Item) {
+func removeItemFromList(item state.Item, itemList []state.Item) ([]state.Item, error) {
```
Comment (antonypap) [inline pkg/infra/game/stage/loot/loot.go]: Error should be returned last

**Pull request closed without merging**